driver/CommonOptions.ml

```diff
@@ -56,6 +70,9 @@ let language_support_options =
 
 let language_support_help =
   {|Language support options (use -fno-<opt> to turn off -f<opt>) :
+  -std=c99  or  -std=c11  or  -std=c18
+                 Choose the ISO C language standard used: C99, C11 or C18.
+                 The "gnu" variants enable some GNU C extensions.
```

Comment:
The line about the gnu variants can probably be removed, since the option does not support the gnu variants.

```suggestion
```

Reply:
Well spotted, thanks!